src/python/WMCore/WebTools/Root.py

```diff
@@ -88,14 +88,30 @@ def access(self):
         response = cherrypy.response
         inheaders = lowerCmsHeaders(request.headers)
         outheaders = response.headers
+        rbytes = 0
+        # identify size of body from HTTP Content-Length header
+        if 'Content-Length' in cherrypy.request.headers:
```

Comment:
Given that you assign `cherrypy.request` to `request`, you can replace all those places like this line and a few others below.